src/main/java/picard/sam/markduplicates/util/AbstractOpticalDuplicateFinderCommandLineProgram.java

```diff
@@ -27,14 +27,15 @@
 import org.broadinstitute.barclay.argparser.Argument;
 import picard.cmdline.*;
 import htsjdk.samtools.util.Log;
+import picard.sam.util.ReadOutputCommandLineProgram;
 
 /**
  * Abstract class that holds parameters and methods common to classes that optical duplicate detection.  We put them here so that
  * the explanation about how read names are parsed is in once place
  *
  * @author Tim Fennell
  */
-public abstract class AbstractOpticalDuplicateFinderCommandLineProgram extends CommandLineProgram {
+public abstract class AbstractOpticalDuplicateFinderCommandLineProgram extends ReadOutputCommandLineProgram {
```

Comment:
hmm `EstimateLibraryComplexity` inherits this but doesn't use it.  I cant see an obvious way to get only things that would use to inherit it but I'll see what I can do